Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -17,25 +17,19 @@
           height="44"
           alt="B.C. Government Logo"
         />
-      </router-link>
+          <b-navbar-brand>
+      <h4 class="nav-link" to="/timeLine">HealthGateway</h4>
     </b-navbar-brand>
-    <b-navbar-brand>
-      <h3>HealthGateway</h3>
+      </router-link>
     </b-navbar-brand>
+  
 
     <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
 
     <!-- Navbar links -->
     <b-collapse id="nav-collapse" is-nav>
       <!-- Menu -->
-      <b-navbar-nav v-if="displayMenu">
-        <router-link class="nav-link" to="/timeLine">
-          <span class="fa fa-stream"></span> Timeline
-        </router-link>
-        <router-link class="nav-link" to="/profile">
-          <span class="fa fa-user"></span> Profile
-        </router-link>
-      </b-navbar-nav>
+ 
       <b-navbar-nav v-if="displayRegistration">
```

Comment:
I dont think we need the registration one anymore since "signin" should directly put you on the registration page.